types/src/lib.rs

```diff
@@ -68,6 +77,8 @@ pub use deploy_info::DeployInfo;
 pub use execution_result::{
     ExecutionEffect, ExecutionResult, OpKind, Operation, Transform, TransformEntry,
 };
+#[cfg(any(feature = "std", test))]
+pub mod file_utils;
```

Comment:
This should be up with the other `mod`s, immediately before `mod gas;`

Reply:
Moved